backend/tests/src/lib/helpers/random/get-random-integer/get-random-integer.helper.ts

```diff
@@ -0,0 +1,11 @@
+const getRandomInteger = (min: number, max: number): number => {
+  const exclusiveMax = max + 1;
+  const maxFromZero = exclusiveMax - min;
+  const random = Math.random();
+  const randomFromZero = Math.floor(maxFromZero * random);
+  const randomFromMin = min + randomFromZero;
+
+  return randomFromMin;
```

Comment:
Very difficult to read. I would prefer this
```suggestion
  return min + Math.floor((max + 1 - min) *  Math.random());
```

Reply:
Why it is hard to read my version? I think the one-liner is even more difficult to read (the algorithm is unclear).
Is it about variable naming? I can rename them. Which names will be better?